apps/davi/src/stores/types.ts

```diff
@@ -26,8 +27,13 @@ export interface FetcherHooksInterface {
     daoId: string,
     proposalId?: `0x${string}`
   ) => ReturnType<typeof useTotalLocked>;
+  useVotingPowerOf: (useVotingPowerOfProps: {

```

Comment:
Do we need to type it as an object with `useVotingPowerOfProps: { params }` ?

It might be better to just pass the params, like `useTotalLocked`. But might be out of scope of this PR...